test/helpers/Staker.handler.sol

```diff
@@ -7,8 +7,8 @@ import {StdUtils} from "forge-std/StdUtils.sol";
 import {console} from "forge-std/console.sol";
```

Comment:
*arbitrary comment location*

File not showing in the diff, but missed one in `DepositIdSet.sol`

Reply:
Done: https://github.com/withtally/staker/pull/121/commits/aaf9160d3f71842b7520ee49387b2c6a01c9f669